ansys/dpf/core/dpf_operator.py

```diff
@@ -107,6 +115,17 @@ def _add_sub_res_operators(self, sub_results):
             method2 = functools.partial(bound_method, name=result_type["operator name"])
             setattr(self, result_type["name"], method2)
 
+    @property
+    @version_requires("3.0")
+    def progress_bar(self) -> bool:
+        """With this property, the user can choose to print a progress bar when
+        the operator's output is requested, default is False"""
+        return self._progress_bar
+
+    @progress_bar.setter
+    def progress_bar(self, value) -> bool:
```

Comment:
Setters shouldn't return a type hint other than `None`, but rather should have one for the argument.

```suggestion
    def progress_bar(self, value: bool) -> None:
```

See discussion:
https://github.com/python/mypy/issues/3004